### PR TITLE
[web] enable ios safari screenshot tests

### PR DESCRIPTION
**Pull request by nturgut**
Enabing ios-safari screenshot tests on LUCI.

Changes the phone used to iPhone 11.

This PR needs to be merged first, please review: https://github.com/flutter/goldens/pull/111

Fixes: https://github.com/flutter/flutter/issues/65670

### CODE REVIEW

File: lib/web_ui/dev/goldens_lock.yaml [hunk 1]
```diff
@@ -1,2 +1,2 @@
-repository: https://github.com/flutter/goldens.git
-revision: b6efc75885c23f0b5c485d8bd659ed339feec9ec
+repository: https://github.com/nturgut/goldens.git
```
Comment (nturgut) [inline lib/web_ui/dev/goldens_lock.yaml:1]: added to test this PR: https://github.com/flutter/goldens/pull/111

Comment (nturgut) [reply lib/web_ui/dev/goldens_lock.yaml]: Changed this to flutter repo, yet we still need to change the has once the goldens PR is in.

File: lib/web_ui/dev/test_platform.dart [hunk 2]
```diff
@@ -197,7 +197,9 @@ class BrowserPlatform extends PlatformPlugin {
         'golden_files',
       );
     } else {
-      await fetchGoldens();
+      if(!isLuci || !Platform.isMacOS) {
```
Comment (yjbanov) [inline lib/web_ui/dev/test_platform.dart:200]: Let's leave a comment explaining why these are special-cased.

Also nit: add one space between `if` and `(`.

**Review by yjbanov — APPROVED** [review]

File: lib/web_ui/dev/test_platform.dart (hunk 2 above)
Comment (nturgut) [reply lib/web_ui/dev/test_platform.dart]: done.

File: lib/web_ui/dev/goldens_lock.yaml (hunk 1 above)
Comment (nturgut) [reply lib/web_ui/dev/goldens_lock.yaml]: made one more change for this. Hopefully they are the correct files both for chrome and ios-safari this time.

**Pull request merged**